src/components/SingleProductView.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { fetchSingleProduct } from '../redux/reducers/product';
+import store from '../redux/store';
+import { SPVFunctional } from './index';
+
+class SingleProductView extends Component {
```

Comment:
Again, linting standards, React.Component someplaces Component others, be a stickler, defend your lint